main.py

```diff
@@ -0,0 +1,56 @@
+import yaml
+import logging
+
+from power_consumption.data_processor import DataProcessor
```

Comment:
Nice!

So you managed to package and import directly from the python module?

Reply:
yes it worked 👍